SPECS/python-rpm-spec/python-rpm-spec.spec

```diff
@@ -0,0 +1,56 @@
+# sitelib for noarch packages, sitearch for others (remove the unneeded one)
+%{!?__python2: %global __python2 %__python}
```

Comment:
Please remove all references to Python 2 - it's been deprecated and we're never going to use it again.

Reply:
Done